test/common/validate_sso_config.go

```diff
@@ -0,0 +1,72 @@
+package common
+
+import (
+	goctx "context"
+	"fmt"
+	"github.com/headzoo/surf"
+	brow "github.com/headzoo/surf/browser"
+	integreatlyv1alpha1 "github.com/integr8ly/integreatly-operator/apis/v1alpha1"
+	"github.com/keycloak/keycloak-operator/pkg/apis/keycloak/v1alpha1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func TestSSOconfig(t TestingTB, ctx *TestingContext) {
+
+	// Validate CPU value requested by SSO
+	t.Log("Validating CPU resources requested for pods")
```

Comment:
I would take this log out, in general, logs should be only covering major steps within a test to highlight a successful/fail outcome.